DeployCryptoBlocker.ps1

```diff
@@ -95,32 +125,29 @@ if ($majorVer -ge 6)
 else
 {
     # Assume Server 2003
-    Write-Host "Other version of Windows detected! Quitting.."
+    Write-Host "Unsupported version of Windows detected! Quitting.."
     return
 }
 
+
 $fileGroupName = "CryptoBlockerGroup"
 $fileTemplateName = "CryptoBlockerTemplate"
 $fileScreenName = "CryptoBlockerScreen"
 
+# Download list of CryptoLocker file extensions
 $webClient = New-Object System.Net.WebClient
 $jsonStr = $webClient.DownloadString("https://fsrm.experiant.ca/api/v1/get")
-$monitoredExtensions = @(ConvertFrom-Json20($jsonStr) | % { $_.filters })
-
+$monitoredExtensions = @(ConvertFrom-Json20 $jsonStr | ForEach-Object { $_.filters })
 
-If (TestPath .\SkipList.txt) {
-    $exclusions = Get-Content .\SkipList.txt
-    ForEach ($exclusion in $exclusions) {
-        if ($exclusion) {
-            $monitoredExtensions = $monitoredExtensions.replace($exclusion.trim(),$null)
-        }
-    }
+If (TestPath .\SkipList.txt)

```

Comment:
This is actually my bad, but can you update this to "Test-Path"; not sure how I didn't catch that in my own merge.